src/sparseml/transformers/__init__.py

```diff
@@ -20,12 +20,17 @@
 
 import logging as _logging
 
+from packaging import version
+
 
 try:
     import transformers as _transformers
 
-    # triggers error if neuralmagic/transformers is not installed
-    assert _transformers.NM_INTEGRATED
+    # triggers error if the latest neuralmagic/transformers is not installed
+    assert (
+        _transformers.NM_INTEGRATED
+        and str(version.parse(_transformers.__version__)) == "4.23.1"
+    )
```

Comment:
optional nit: for debugging purposes it may be easier to separate this into two separate asserts
```suggestion
    assert _transformers.NM_INTEGRATED
    assert str(version.parse(_transformers.__version__)) == "4.23.1", _transformers.__version__
```